ocfweb/main/templates/main/home.html

```diff
@@ -7,11 +7,24 @@
 {% block hero %}
 <div class="ocf-penguin-swing"></div>
 
-<div class="ocf-hero">
+<div class="ocf-hero" style="background-color:#888">
     <div class="container">
         <div class="row">
             <div class="col-sm-8">
-                <h1>Welcome to the Open Computing Facility!</h1>
+                <h1>The OCF Stands with Black Lives Matter</h1>
+                <p>We at the OCF believe in the idea of freedom. Both free technology and free people, regardless of color, gender, or sexuality. Here are some links to some resources.</p>
+                <ul>
+                    <li><a href="https://docs.google.com/document/d/1H-Vxs6jEUByXylMS2BjGH1kQ7mEuZnHpPSs1Bpaqmw0/mobilebasic" style="color:black">Justice in June</a></li>
+                    <li><a href="https://docs.google.com/document/u/1/d/1nkX6-WMWCgcIOV5v_q0xNF0MQ4_dvyUol1CnYh1RlHE/mobilebasic" style="color: black">BLM Resources for U.C. Students</a></li>
+                    <li><a href="https://blacklivesmatters.carrd.co/" style="color:black">Ways You Can Help</a></li>
+                    <li><a href="https://blacklivesmatter.com/resource-category/toolkits/" style="color:black">BLM Toolkits</a></li>
+                    <li><a href="https://www.aclu.org/issues/racial-justice/race-and-criminal-justice/blacklivesmatter" style="color:black">ACLU BLM</a></li>
+                </ul>
+            </p>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-sm-8">
                 <p>The Open Computing Facility (OCF) is an all-volunteer student organization dedicated to free computing for all University of California, Berkeley students, faculty, and staff.</p>
                 <p class="ocf-button-holder">
                 <a href="{% url 'about-staff' %}" class="btn btn-lg">Learn what we do</a>
```

Comment:
If there's a way to make these buttons smaller I would be for it. Right now it is de-emphasizing your message in the above.

Reply:
You could also change the color of the buttons.

Reply:
Basically right now they just pop out too much compared to the text, and the buttons don't have anything to do with the text

Reply:
Yeah I think lightening it is a good idea. I also made the buttons smaller, but changing the color didn't really look right imo.

Reply:
this is what the updates look like.
![Screen Shot 2020-06-18 at 8 49 12 PM](https://user-images.githubusercontent.com/35557295/85095064-35e2d600-b1a5-11ea-97b8-7c6673ab7839.png)